src/pages/AddressBook.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <modal-page
+    v-if="addressBook.length"
+    class="address-book"
+    title="Æddress Book"
+    :redirectToOnClose="{ name: 'apps' }"
+  >
+    <ae-divider />
+    <template v-for="(c, idx) in addressBook">
+      <address-book-item
+        :name="c.name"
+        :address="c.address"
+        @click="openIdx = openIdx === idx ? -1 : idx"
+      >
+        <ae-icon slot="icon" name="chevron" :rotate="idx === openIdx ? -90 : 90" />
+      </address-book-item>
+      <div v-if="idx === openIdx" class="actions">
+        <router-link :to="{ name: 'transfer', params: { to: c.address } }">
+          <ae-app-icon src="static/icons/notary.svg" />
+        </router-link>
+      </div>
+      <ae-divider />
+    </template>
+
+    <fixed-add-button :to="{ name: 'address-book-new' }" />
```

Comment:
since this button is in the if block it wont be shown when there are no contacts which prevents the user from adding one in the first place